ui/raidboss/data/07-dt/hunts/living_memory.ts

```diff
@@ -395,7 +402,21 @@ const triggerSet: TriggerSet<Data> = {
         return output.protean!();
       },
       outputStrings: {
-        protean: 'Protean (x2)',
+        protean: 'Protean (then dodge into)',
+      },
+    },
+    {
+      id: 'Hunt Mica the Magical Mu Double Misdirect Move',
+      type: 'Ability',
+      netRegex: { id: '9727', source: 'Mica the Magical Mu', capture: false },
+      suppressSeconds: 1,
+      infoText: (_data, _matches, output) => {
+        return output.moveInto!();
+      },
+      outputStrings: {
+        moveInto: {
+          en: 'Dodge',
+        },
       },
```

Comment:
```suggestion
      response: Responses.moveAway('alert'),
```

Reply:
Suggest alert severity here simply to give an alternating cadence to the initial/follow-up triggers.